tests/README.md

```diff
@@ -0,0 +1,27 @@
+# OpsCenter SQL tests
+
+## Running tests
+
+Place the `parameters.py` file in the `tests/sql` directory, with the connection information:
```

Comment:
It would be better to use the same technique as `deploy/deploy.py` read ~/.snowsql/config

Reply:
agree and will fix.